app/src/metrics.ts

```diff
@@ -86,8 +86,8 @@ export function getMetricCalcs(
       (blob: HydratedGitBlobObject, cache: MetricCache) => {
         if (!cache.legend) {
           cache.legend = [
-            commit.minNoCommits,
-            commit.maxNoCommits,
+            `${commit.minNoCommits}`,
```

Comment:
Why does this use a string?

Reply:
The interface for gradient legend labels(or what they are called), is changed to string such that coldmap can display a date. Heatmap legend label therefore is cast to string.

I feel the cast isn't too ugly. What do you think?